app/assets/javascripts/blacklight/core.js

```diff
@@ -14,7 +14,8 @@ Blacklight = function() {
 }();
 
 // turbolinks triggers page:load events on page transition
-// If app isn't using turbolinks, this event will never be triggered, no prob. 
+// If app isn't using turbolinks, this event will never be triggered, no prob.
+// Support for Turbolinks classic
 $(document).on('page:load', function() {
```

Comment:
Can this just be:

`$(document).on('page:load turbolinks:load', function() {`?